post/models.py

```diff
@@ -18,3 +17,18 @@ def get_absolute_url(self):
     # Print the string representation of Post
     def __str__(self):
         return self.post_story
+
+
+# Based on code by Django Girls, url: https://djangogirls.gitbooks.io/django-girls-tutorial-extensions/homework_create_more_models/
+class Comment(models.Model):
+    post = models.ForeignKey(Post, related_name='comments')
+    author = models.CharField(max_length=80)
```

Comment:
I'm not sure exactly how this would work.. I tried adding the following lines to post/models.py - then ran makemigrations, then migrate. It resulted in the comment author being a drop down selection, rather than the user who is logged in. I'm going to merge the pull request as is into dev as and try to figure that out later on. If you know how to fix it let me know!

from dashboard import models as models_in_dashboard

author= models.ForeignKey(models_in_dashboard.Author, related_name='author')